bucket/robo3t.json

```diff
@@ -1,27 +1,22 @@
 {
-    "version": "1.4.2",
+    "version": "1.4.3",
     "description": "The free lightweight GUI for MongoDB enthusiasts",
     "homepage": "https://robomongo.org",
     "license": "GPL-3.0-only",
-    "architecture": {
-        "64bit": {
-            "url": "https://download.studio3t.com/robomongo/windows/robo3t-1.4.2-windows-x86_64-8650949.zip",
```

Comment:
Unless the tool is 32 bit only, for new updates we prefer retaining architecture and 64bit for 64bit apps to minimize inconvenience for 32 bit users

So please use the old format